### PR TITLE
Update ITEM_en.tsv

### PR DESCRIPTION
**Pull request by emailboxu**
Deleted duplicate line 882 (Chainmail). Not present in KR files and is a duplicate line to line 884.

Hat string changes.

### CODE REVIEW

File: ITEM_en.tsv [hunk 1]
```diff
@@ -1447,9 +1446,9 @@ ITEM_20150317_001548	Shepherd Costume
 ITEM_20150317_001549	Chaplain Costume
 ITEM_20150317_001550	Empty_Hat
 ITEM_20150317_001551	Hair Costume
-ITEM_20150317_001552	Costume for the hair slot. You can equip it by right-clicking the item.
+ITEM_20150317_001552	A Costume for the Hat slot. You can equip it by right-clicking the item.
```
Comment (Sunaries) [inline ITEM_en.tsv]: "_Hair decoration. Right-click to equip._"

Hat is incorrect. The first sentence refers to the item being decoration for the player's hairstyle.

Comment (emailboxu) [inline ITEM_en.tsv]: Thanks, fixed in next update.

**Comment by emailboxu** [comment]
Will continue with Crafting strings tomorrow.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**